common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -552,6 +553,70 @@ public ICacheRecord load(@NonNull final String clientId,
                 null // not applicable
         );
 
+        {
```

Comment:
nit: Since finding refresh tokens is now complicated, should it be in its own (potentially set of) private method(s)?

Reply:
Yeah, I'll take that feedback. This method becoming long

Reply:
Resolved via
- a22f456f20136fe51c96c5c2cf5d786c39b841ba

Tweaked javadoc in:
- 3ab2e13c109012a373bc56cee8ad0f81382d2644